manifests/http_port.pp

```diff
@@ -31,6 +31,7 @@
 #   When set to `true` creates https_port entries. Defaults to `false`.
 # @param order
 #   Order can be used to configure where in `squid.conf`this configuration section should occur.
+
```

Comment:
I think I've seen tools break on this empty line

Reply:
Copy past error, removed it.

Happy to help !